examples/radiotoolbutton.py

```diff
@@ -0,0 +1,40 @@
+from gi.repository import Gtk
+
+from sugar3.graphics.radiotoolbutton import RadioToolButton
+from sugar3.graphics.radiopalette import RadioPalette, RadioMenuButton
+from sugar3.graphics.xocolor import XoColor
+import common
+
+test = common.Test()
```

Comment:
I'm not a big an of common.test, because it makes more sense to just use a Gtk.Window so people can copy the code.

So just something like

    window = Gtk.Window()
    window.show()
    window.connect('destroy', Gtk.main_quit)

then add the box to window

and instead of common.main, just run `Gtk.main()`